core/repository/manager/src/main/java/org/eclipse/rdf4j/repository/manager/RepositoryManager.java

```diff
@@ -537,19 +430,29 @@ public RepositoryInfo getRepositoryInfo(String id) throws RepositoryException {
 		return null;
 	}
 
-	public Collection<RepositoryInfo> getAllRepositoryInfos() throws RepositoryException {
-		return getAllRepositoryInfos(false);
-	}
+	/**
+	 * Retrieve meta information of all configured repositories.
+	 * 
+	 * @return a collection of {@link RepositoryInfo} objects
+	 * @throws RepositoryException if the repository meta information could not be retrieved.
+	 */
+	public abstract Collection<RepositoryInfo> getAllRepositoryInfos() throws RepositoryException;
 
+	/**
+	 * @deprecated since 4.0 - use {@link #getAllRepositoryInfos()} instead.
+	 */
+	@Deprecated
 	public Collection<RepositoryInfo> getAllUserRepositoryInfos() throws RepositoryException {
-		return getAllRepositoryInfos(true);
+		return getAllRepositoryInfos();
 	}
 
 	/**
-	 * @param skipSystemRepo
-	 * @throws RepositoryException When not able to retrieve existing configurations
+	 * @deprecated since 4.0 - use {@link #getAllRepositoryInfos()} instead.
 	 */
-	public abstract Collection<RepositoryInfo> getAllRepositoryInfos(boolean skipSystemRepo) throws RepositoryException;
+	@Deprecated
+	public Collection<RepositoryInfo> getAllRepositoryInfos(boolean skipSystemRepo) throws RepositoryException {
+		return getAllRepositoryInfos();
```

Comment:
Yes you did :)

Btw. `@Deprecated` can take arguments now: https://docs.oracle.com/javase/9/core/enhanced-deprecation1.htm#JSCOR-GUID-E182DAF6-282B-4C1F-8CCF-0A61008E3F11